### ISSUE TITLE
Error unboxing chat message

### ISSUE THREAD

**Issue by ajsb85** [keybase/client#10520]
`Error unboxing chat message: You are not a member of this team (error 2623)`

![image](https://user-images.githubusercontent.com/663460/36102570-fe843ee2-100c-11e8-8f3b-e10acbc3432d.png)

**Comment by ajsb85** [comment]
![image](https://user-images.githubusercontent.com/663460/36102619-261cd068-100d-11e8-95d4-f3c83c10a53d.png)

**Comment by ajsb85** [comment]
![image](https://user-images.githubusercontent.com/663460/36102634-2fc7b93e-100d-11e8-9cf8-d1648813ac36.png)

**Comment by ajsb85** [comment]
![image](https://user-images.githubusercontent.com/663460/36102644-355d98be-100d-11e8-843c-dc4172bbd246.png)

**Comment by maxtaco** [comment]
Cc @mmaxim does this look right?

**Comment by mmaxim** [comment]
Is this still a problem, if so, can you try `keybase db nuke`?

**Comment by ajsb85** [comment]
`keybase db nuke` done

Still the same problem.
I had this problem from a lot of months ago.
I was trying to use keybase to see if it already fixed.

Regards.

**Comment by mmaxim** [comment]
What version of Keybase are you running, can you send a log?

**Comment by mmaxim** [comment]
`keybase log send` to send a log

**Comment by mmaxim** [comment]
Actually I think I see what is going on here. @maxtaco this might end up being a little tricky. What has happened is the following:

1.) `cyrildamas` started an implicit team chat with `ajsb85` when `ajsb85` had no PUK.
2.) `cyrildamas` takes all their devices offline. `ajsb85` comes back and wants to chat.
3.) `ajsb85` just sees errors since there is no one online to sign his invite into the team.

**Comment by ajsb85** [comment]
my log id: a6e93ab971e0e898037f841c

**Comment by ajsb85** [comment]
keybase version 1.0.40-20180205105125+493a528

**Comment by mmaxim** [comment]
@ajsb85 you can probably get this situation fixed if cyrildamas starts a Keybase client again.

**Comment by ajsb85** [comment]
We are coworkers.
And this issue is from many months ago.
I reset my keybase and still the same problem.

**Comment by ajsb85** [comment]
He shared a new information with me recently. 
Could you help us? @cyrildamas

**Comment by ajsb85** [comment]
![image](https://user-images.githubusercontent.com/663460/36219855-0997ef38-11b9-11e8-8f3e-8138f879e134.png)

**Comment by mmaxim** [comment]
Does he have a device online?

**Comment by ajsb85** [comment]
He opened the client and sent again a message 2 minutes ago.

**Comment by maxtaco** [comment]
Can he do `keybase log send`?

**Comment by ajsb85** [comment]
@cyrildamas please exec `keybase log send`

**Comment by ajsb85** [comment]
We are using this thread as chat 🤣

**Comment by cyrildamas** [comment]
my log ID  053857c38194803a1b9db01c

**Comment by ajsb85** [comment]
![image](https://user-images.githubusercontent.com/663460/36220253-373e7802-11ba-11e8-9ea8-2a919952ada4.png)

**Comment by mmaxim** [comment]
@cyrildamas sorry to do this to you, but could you log send one more time?

**Comment by ajsb85** [comment]
![image](https://user-images.githubusercontent.com/663460/36220364-93e612f4-11ba-11e8-936e-1f19c81a4fac.png)

**Comment by cyrildamas** [comment]
![capture d ecran 2018-02-14 a 19 06 08](https://user-images.githubusercontent.com/12440165/36220338-7e77db3c-11ba-11e8-8b18-ab94cfbb8994.png)

**Comment by ajsb85** [comment]
@cyrildamas please exec again `keybase log send`

**Comment by cyrildamas** [comment]
Success! Your log ID is:  6d029d85d1ec62af4bbdad1c

**Comment by ajsb85** [comment]
Success! Your log ID is:

  509a813510174a3b062f4d1c

**Comment by cyrildamas** [comment]
my MacOS client Version 1.0.40-20180205183337+269845275

**Comment by maxtaco** [comment]
@ajsb85 we need to be better about communicating this, but by resetting your account, you made things worse.  In general, reset accounts are challenging to handle, and have more bugs than non-reset accounts.

**Comment by mmaxim** [comment]
So I made a change on the server which may have fixed this for you all. Both sides of the chat should now advertise ajsb85 as being reset, and cyrildamas should have the option to "re-add" him into the conversation.

**Comment by ajsb85** [comment]
![image](https://user-images.githubusercontent.com/663460/36229773-a8203588-11d8-11e8-8c67-7c00a3d0178e.png)

**Comment by mmaxim** [comment]
Yep that is how it should show up for you, cyrildamas should be able to let you back in.

**Comment by cyrildamas** [comment]
![capture d ecran 2018-02-15 a 09 11 18](https://user-images.githubusercontent.com/12440165/36246816-c43197a4-1231-11e8-9e02-2f54f176b32a.png)
when i "Let them In" =
![capture d ecran 2018-02-15 a 09 11 25](https://user-images.githubusercontent.com/12440165/36246859-fb5137e4-1231-11e8-8592-613e3cc0f671.png)

**Comment by mmaxim** [comment]
Sorry about that guys, this is a bug in the client adding back a reset user that did not have a per user key when the conversation was created. I am going to try and fix it right now.

**Comment by ajsb85** [comment]
![image](https://user-images.githubusercontent.com/663460/37236193-9cc560d4-2405-11e8-8b97-04026816af5e.png)


![image](https://user-images.githubusercontent.com/663460/37236158-4b6a22a6-2405-11e8-9f49-9cba578ce7c3.png)

**Comment by ajsb85** [comment]
![image](https://user-images.githubusercontent.com/663460/37259425-68f8c930-2586-11e8-99d0-af61693806e6.png)

**Comment by mmaxim** [comment]
Great!

**Comment by wesinator** [comment]
I'm experiencing a similar issue on macOS
`Version 2.12.6-20190103193723+07470de987 (2.12.6-20190103193723+07470de987)`

I started being followed by team members and was shared chats before I was a member of the team. The chats shared with me display `Unable to decrypt chat message: You are not a member of this team (error 2623)`

After running `keybase team request-access` and getting welcomed to the team, I still see the unable to decrypt error message on the most recent messages, but I can read older messages from over a month ago.

<img width="504" alt="image" src="https://user-images.githubusercontent.com/5124946/51130248-9163a400-17fa-11e9-8b96-daebd8fab533.png">

I also see this error message in a direct conversation with a user on my team, making me unable to read the chats they sent me.

I don't have any keys set up yet on my account, but if that were the issue, I would think it should show a different error message, and I wouldn't be able to read older messages.
